examples/fabric-admin/main.cpp

```diff
@@ -18,14 +18,43 @@
 
 #include <commands/clusters/SubscriptionsCommands.h>
 #include <commands/common/Commands.h>
+#include <commands/fabric-sync/Commands.h>
 #include <commands/interactive/Commands.h>
 #include <commands/pairing/Commands.h>
 #include <zap-generated/cluster/Commands.h>
 
 #include <iostream>
 #include <string>
+#include <thread>
```

Comment:
This is a new dependency on `Thread`. Suggest using system layer timer and separate handling of events to avoid the ApplicationInit being an oddly blocking pattern.

Reply:
acked